### PR TITLE
Adding expyplot for using matplotlib.pyplot

### PR DESCRIPTION
**Pull request by cdesch**
## Title

Add Package "expyplot"

## Description

Adding new package

## Commit message

Expyplot allows you to use 'all' of the functions in matplotlib.pyplot (in reality, there are some that I left out because they don't make any sense in this context, or are deprecated).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1033,6 +1033,7 @@ There are [other sites with curated lists of elixir packages](#other-awesome-lis
 * [exfcm](https://github.com/Hajto/ExFCM) - Simple wrapper for posting Firebase Cloud Messages.
 * [exldap](https://github.com/jmerriweather/exldap) - A module for working with LDAP from Elixir.
 * [exlibris](https://github.com/pragdave/exlibris) - A collection of random library functions.
+* [expyplot](https://github.com/MaxStrange/expyplot) - Elixir interface for Plotting/Graphing library using matplotlib.pyplot
```
Comment (simonewebdesign) [inline README.md:1036]: Please could you add a dot at the end of the sentence?

Comment (cdesch) [reply README.md]: All Set

**Comment by simonewebdesign** [comment]
Sorry, can you also make sure it's ordered alphabetically please?

**Comment by cdesch** [comment]
My Mistake. I didn't look at all of the letters :-( -- fixed

**Comment by simonewebdesign** [comment]
Thank you!

**Pull request merged**